_posts/2020-06-02-runtime-dependencies.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Runtime dependencies in Copr
+author: Dominik Tureček
+layout: post
+---
+
+In the last release, we've added support for runtime dependencies in Copr. If your project needs another package to run but you don't need said package as a part of your project, you can have it as a runtime dependency instead. You can now specify other repositories (either Copr projects or external) as runtime dependencies. These repositories will then be enabled alongside your Copr project.
+
+We are currently testing the feature and are looking for users' feedback - please share any issues that you might find. Later, once this feature is tested, we plan to generalise it and add it to dnf and createrepo_c as well.
```

Comment:
generalise is British variant, generalize is US variant. Are you intentionally using British English?

Reply:
Not intentional, although I prefer the British spelling so it sometimes slips in.